src/purchase-bill/dto/create-purchase-bill.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsNotEmpty } from "class-validator";
+export class CreatePurchaseBillDto {
+
+  @IsNotEmpty()
+  @ApiProperty()
+  totalCost: number;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  advancePaid: number;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  tireQuantity: number;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  costPaid: number;
+
+  @ApiProperty()
+  id: string;
+
```

Comment:
we don't need an id while we create something.

Reply:
It throws the error:

> 
>    Types of property 'vendor_id' are incompatible.
>         Type 'string' is not assignable to type 'never'.

Reply:
When I was doing this :

vendor: {
connect: {
id: vendor_id 
}
}
it was throwing error. 

So I changed that by mapping directly to vendor_id